test/kata2.booleanToWord.test.js

```diff
@@ -1,5 +1,13 @@
 const { booleanToWord } = require("../src");
 
 describe("booleanToWord", () => {
-  // how do we create specs again???
+  it('returns the stirng Yes when passed as true', () => {
+
```

Comment:
These tests are great. You can't do more than one test in this one because there are only two possible inputs and you have tested both.